variables.tf

```
@@ -0,0 +1,186 @@
+################################################################################
+# General
+################################################################################
+variable "name" {
+  description = "Name to be used on all the resources as identifier"
+  type        = string
+  default     = ""
+}
+
+variable "tags" {
+  description = "A list of tags to add to all resources"
+  type        = list(string)
+  default     = []
+}
+
+variable "vpc_tags" {
+  description = "Additional tags for the VPC"
+  type        = list(string)
+  default     = []
+}
+
+variable "timeouts" {
+  description = "Define maximum timeout for creating, updating, and deleting VPC resources"
+  type        = map(string)
+  default     = {}
+}
+
+variable "azs" {
```

```suggestion
variable "zones" {
```

`azs` are not that easy to understand compared to `zones`